src/policy/mod.rs

```diff
@@ -42,3 +43,9 @@ pub const DEFAULT_BYTES_PER_SIGOP: u32 = 20;
 /// The minimum feerate, in sats per kilo-virtualbyte, for defining dust. An output is considered
 /// dust if spending it under this feerate would cost more in fee.
 pub const DUST_RELAY_TX_FEE: u32 = 3000;
+
+/// The virtual transaction size, as computed by default by bitcoind node.
+pub fn get_virtual_tx_size(weight: i64, n_sigops: i64) -> i64 {
+    (cmp::max(weight, n_sigops * DEFAULT_BYTES_PER_SIGOP as i64) + WITNESS_SCALE_FACTOR as i64 - 1)
```

Comment:
Idk enough about core, but it appears to have a more general version of this function where `DEFAULT_BYTES_PER_SIGOP` is variable. Is this actually never used?

Reply:
It's used in a rather convoluted manner in Bitcoin Core for the sigop cost to be configurable. We only care about the default here, which is what 99.9% of the network nodes run.